src/commands/user/quests.js

```diff
@@ -23,9 +23,9 @@ module.exports = {
 		
 		const questSession = new Quest(client, reply)
 		await questSession.start(sessionID, user, locale, messageRef)
-		if (questSession.getSessionActive) return 
```

Comment:
Is this temporary for testing?

Reply:
Yes, lines 26 and 27 should be uncommented, they were commented for testing reasons